### PR TITLE
Feat/add manufacturer filter toggle to lts assistant

### PR DESCRIPTION
**Pull request by ivladu-plenty**
[AB#136989](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/136989)
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [ ] Changes to SCSS have been accounted for in plentyShop LTS Modern

@plentymarkets/plentyshop

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -956,6 +956,64 @@
                     "options": {
                         "defaultValue": "false"
                     }
+                },
+                "item.manufacturer_data": {
+                    "type": "multiCheckBox",
+                    "required": false,
+                    "label": "Config.itemManufacturerDataLabel",
+                    "options": {
+                        "defaultValue": "",
```
Comment (stentrop) [inline config.json]: what is with default value ? 

should be selected all

File: config.json [hunk 2]
```diff
@@ -956,6 +956,64 @@
...
+                            {
+                                "value": "street",
+                                "caption": "Config.itemManufacturerStreet"
+                            },
+                            {
+                                "value": "houseNo",
+                                "caption": "Config.itemManufacturerHouseNo"
+                            },
+                            {
+                                "value": "zip",
```
Comment (stentrop) [inline config.json]: value should be "postcode"

File: src/Wizard/ShopWizard/Steps/Builder/DisplayedInformationStep.php [hunk 3]
```diff
@@ -195,6 +198,14 @@ private function generateItemView(): array
                         "name" => "Wizard.attributeSelectDefaultOption",
                     ]
                 ],
+                "displayInfo_manufacturerInformation" => [
+                    "type" => "checkboxGroup",
+                    "defaultValue" => [],
```
Comment (stentrop) [inline src/Wizard/ShopWizard/Steps/Builder/DisplayedInformationStep.php]: same ? why not a default value ?

**Review by stentrop — DISMISSED** [review]

File: config.json (hunk 1 above)
Comment (ivladu-plenty) [reply config.json]: done

File: config.json (hunk 2 above)
Comment (ivladu-plenty) [reply config.json]: done

File: src/Wizard/ShopWizard/Steps/Builder/DisplayedInformationStep.php (hunk 3 above)
Comment (ivladu-plenty) [reply src/Wizard/ShopWizard/Steps/Builder/DisplayedInformationStep.php]: done

**Pull request closed without merging**